share-link.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+   <head>
+      <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+   </head>
+<div>
+   
+   <div class="share-buttons ">
+                    <a href="# " class="fab fa-facebook" style="color:#6D1AD8;"></a>
+                    <a href="# " class="fab fa-twitter" style="color:#6D1AD8;"></a>
+
+                </div>
+         
+       
+    </div>
+    <script>
+        let facebook_link = document.querySelector('.fa-facebook')
+        facebook_link.href = `https://www.facebook.com/sharer/sharer.php?u=${window.location.href}`
+
+        let twitter_link = document.querySelector('.fa-twitter')
+        let cert_title = document.querySelector('#cert-title').innerHTML
```

Comment:
There is no element with an id="cert-title". Can you explain the workflow of this twitter share link? I understand